pose_collector:
  clock: 100
virtual_drone_clock: 100
field_cbf: [[-1.3, 1.3], [-1.3, 1.3]] # 衝突回避で避けられるよう、psiよりも一回り大きくとる
central_clock: 20
agents:
  agent_manager_clock: 20
  u_max: 0.3
  camera_sensor_rad: 0.7895822394
  kp_z: 1
  ref_z: 1
  kp_yaw: 1
  ref_yaw: 0
  unom_max: 0.2
grape_detector:
  grape_positions: [[0.5, 0.5, 0], [-0.5, -0.5, 0]]
  detect_radius: 0.3 ### = height * tan(math.pi/2 - range[3][0])
  target_field_radius: 0.3 ### = height * tan(math.pi/2 - range[3][0])
  clock : 1
  size: [0.1, 0.1, 0.1]
  grape_span: 0.5
  pictogram:
    size: 0.5
    character: fa-apple-alt
trees:
  xy: [[50, 50]]
  avoid_radius: [0.5]
  no_phi_radius: [0.5]